src/LayerPlan.cpp

```diff
@@ -774,6 +774,31 @@ void LayerPlan::writeGCode(GCodeExport& gcode)
                 }
                 gcode.writeTemperatureCommand(prev_extruder, prev_extruder_temp, wait);
             }
+
+            if (!getPrimeTowerIsPlanned())
```

Comment:
If you look at the photo(s) posted on the forum thread by the person who is kindly testing the code, it does look rather messy when doing the prime before the move. But is that worse than leaving a big blob on the print?